app/models/birthday_email_template.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class BirthdayEmailTemplate < ApplicationRecord
+  validates :body, :name, :title, presence: true
+
+  def next
+    BirthdayEmailTemplate.where('id > ?', id)[0]
+  end
+
+  def set_last_used
+    return if last_used
+
+    last_used_templates = BirthdayEmailTemplate.where(last_used: true)
```

Comment:
maybe we could just do:
BirthdayEmailTemplate.update_all(last_used: false)
update!(last_used: true)